frontend/sw360-portlet/src/main/webapp/component/searchfield.jspf

```diff
@@ -1,4 +1,16 @@
-<div id="component-searchField" class="content1">
+<%--
+ ~ Copyright COPYRIGHT HOLDER, 2017.
```

Comment:
right, at some point pls do not forget to replace all the place holder occurrences with "Siemens AG 2017"